asyncio_mqtt/client.py

```diff
@@ -78,16 +117,27 @@ def __init__(
             )
 
     @property
-    def id(self):
+    def id(self) -> str:
         """Return the client ID.
 
         Note that paho-mqtt stores the client ID as `bytes` internally.
         We assume that the client ID is a UTF8-encoded string and decode
         it first.
         """
-        return self._client._client_id.decode()
+        return cast(bytes, self._client._client_id).decode()
 
-    async def connect(self, *, timeout=10):
+    @property
+    def _pending_calls(self) -> Set[int]:
```

Comment:
I'd personally prefer a generator for this. Something like:
```python
yield from self._pending_subscribes.keys()
yield from self._pending_unsubscribes.keys()
yield from self._pending_publishes.keys()
```

Accomplishes the same thing but:
 * Is shorter
 * Doesn't allocate memory for a set

In any case, a `Set` will do fine as well. 👍

Reply:
Ah yes, that's a nice solution.